src/AirshipBindings.Portable/AirshipBindings.Portable.Android/Push/TagEditor.cs

```diff
@@ -0,0 +1,18 @@
+﻿/*
+ Copyright 2017 Urban Airship and Contributors
+*/
+
+using System;
+using UrbanAirship;
+
+namespace AirshipBindings.Portable.Push
+{
+	public partial class TagEditor
+	{
+		public TagEditor Clear()
+		{
+			UAirship.Shared().PushManager.EditTags().Clear();
```

Comment:
We need to avoid doing this until apply is called. I think on Android we store a clear flag and apply the clear first then do any adds